### ISSUE TITLE
Select Audio Output Device

### ISSUE THREAD

**Issue by tedmoore** [tedmoore/serge-archive-oF#9]
Currently there is no way to select the audio output device, either when the application opens, or while in use. 

It would be great to have a menu to change the audio output device.

**Comment by jeanbrazeau** [comment]
I'm thinking a small icon in the top right corner to pull up a menu?... menu tbd

<img width="669" alt="Screen Shot 2023-03-15 at 11 04 04 PM" src="https://user-images.githubusercontent.com/7369279/225529265-4fa10972-6f4c-4437-8ca5-8742a27370b6.png">
<img width="501" alt="Screen Shot 2023-03-15 at 11 02 17 PM" src="https://user-images.githubusercontent.com/7369279/225529274-9c694c6d-6e53-468c-90ba-f21bbf9e7aa9.png">

**Comment by tedmoore** [comment]
Yeah, I think the icon is excellent, however that position might not always be empty space on the plot, so the icon might be covering up some data points! Where else might it go? Maybe to the left of the TKB along side the midi button and recording switch?
